src/main/java/org/cascadebot/cascadebot/utils/buttons/PersistentButtonGroup.java

```diff
@@ -0,0 +1,44 @@
+package org.cascadebot.cascadebot.utils.buttons;
+
+import javax.naming.OperationNotSupportedException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class PersistentButtonGroup extends ButtonGroup {
+
+    private List<PersistentButton> persistentButtons = new ArrayList<>();
+
+    private PersistentButtonGroup() {
+        super(-1, -1, -1);
+    }
+
+    public PersistentButtonGroup(long ownerId, long channelId, long guildId) {
+        super(ownerId, channelId, guildId);
+    }
+
+    @Override
+    public List<Button> getButtons() {
+        return persistentButtons.stream().map(PersistentButton::getButton).collect(Collectors.toList());
+    }
+
+    @Override
+    public void addButton(Button button) {
+    }
+
+    @Override
+    public void removeButton(Button button) {
```

Comment:
If these are not supposed to be used, they should probably throw an exception inside to make debugging easier. Something like `UnsupportedOperationException` would work perfect in this case

Reply:
I tried but it wanted me to add a throw clause to the method and therefor the parent method which broke everything.

Reply:
If you use `java.lang.UnsupportedOperationException` it won't force you to try catch since it's a runtime exception. I think before you tried using the javax one which is probably a checked exception

Reply:
I'll try that